kraken-build/.changelog/_unreleased.toml

```diff
@@ -0,0 +1,6 @@
+[[entries]]
+id = "a0755acd-c588-47a7-bd6d-7ade753d1d1e"
+type = "fix"
+description = "Enable cargo:token as a global-credential-provider"
```

Comment:
Could we add the rationale here please? Something along the lines of "this is required for Cargo 0.74.0 and up because .."